hazelcast/src/main/java/com/hazelcast/map/impl/OwnedEntryCostEstimator.java

```diff
@@ -17,17 +17,20 @@
 package com.hazelcast.map.impl;
 
 /**
- * Size estimator general contract.
+ * Owned entry cost estimator general contract.
  *
- * @param <T> the type of object which's size going to be estimated.
+ * @param <K> the type of key which's size going to be estimated.
+ * @param <V> the type of key which's size going to be estimated.
  */
-public interface SizeEstimator<T> {
+public interface OwnedEntryCostEstimator<K, V> {
```

Comment:
Agree with @ahmetmircik, `EntryCostEstimator` fits better, owned or not, it can estimate cost of an entry.